osmdroid-android/src/main/AndroidManifest.xml

```diff
@@ -8,5 +8,8 @@
 	<uses-feature android:name="android.hardware.telephony" android:required="false" />
 	<uses-feature android:name="android.hardware.wifi" android:required="false" />
 
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
i thought we all agreed that not everyone needs this. While is convenient for some developers, it may add too many unneeded permissions in specific use cases

Reply:
I removed the permission to have some progress. But as follow up, 

* lint will show an issue 👎 
* a possible error source is now re-introduced 👎 
* most of the developer have to add this permission manually in there own repo 👎 , instead of a single place here

I would do it vice-versa and go with adding the permission and who don't want this can remove it

`<uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"" tools:node="remove"/>`

## Benefit:
It will work out of the box, nobody is force to do something and it removes a potential error source